dev/com.ibm.websphere.org.eclipse.microprofile.reactive.streams.operators.1.0/bnd.bnd

```diff
@@ -21,8 +21,9 @@ Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
 
 Import-Package: \
   org.reactivestreams; version="[1.0,1.1)", \
+  com.ibm.ws.microprofile.reactive.streams.spi.impl;version=1.0,\
```

Comment:
_*Technically*_, I think this breaks the `com.ibm.websphere.appserver.org.eclipse.microprofile.reactive.streams.operators-1.0` private feature ... not sure if that matters??

Reply:
Would be ok with `resolution:=optional`

Reply:
Done